frontend/talentsearch/src/js/components/workLocationPreferenceForm/workLocationPreferenceOperations.graphql

```diff
@@ -0,0 +1,15 @@
+query WorkLocationPreference{
+  me{
+    id
+    email
+    locationPreferences
+    locationExemptions
+  }
+}
```

Comment:
Oh very cool I didn't know we had this query 😄

Reply:
> Nice work Vanmathy! Looking really good. I just found a few small remaining things.

Thank you! @yonikid15  for the review. I broke my code when I started to make the changes requested by @tristan-orourke . I will do the changes after I fixed that.